app/controllers/clas_controller.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class ClasController < ApplicationController
+  before_action :set_cla, only: %i[show edit update]
+
+  def index
+    @clas = authorize Cla.all
+  end
+
+  def show
+    @cla_version = if params[:version]
+                     @cla.versions.find(params[:version])
+                   else
+                     @cla.current_version
+                   end
+  end
+
+  def display_for_signing
+    @cla = Organization.find_by(login: params[:organization]).repositories.find_by(name: params[:repository]).cla
+  end
+
+  def new
+    @cla = authorize Cla.new
+  end
+
+  def create
+    ActiveRecord::Base.transaction do
```

Comment:
Are you authorizing the creation here? What will happen if a random person on the internet sends `POST` to shis endpoint directly (avoiding the form altogether?)
Is there a test for this?